pkg/controller/controller_broker.go

```diff
@@ -279,7 +303,49 @@ func (c *controller) reconcileServiceBroker(broker *v1alpha1.ServiceBroker) erro
 		// Delete ServiceClasses that are for THIS ServiceBroker.
 		for _, svcClass := range svcClasses {
 			if svcClass.ServiceBrokerName == broker.Name {
-				err := c.serviceCatalogClient.ServiceClasses().Delete(svcClass.Name, &metav1.DeleteOptions{})
+				// TODO : implement and switch to filter
+				// selectors. This is horrifyingly inefficient
```

Comment:
Are you planning to implement field selection in this PR?  I would be okay with it in this PR or an immediate follow-up.

Reply:
Did you want to reply to this?

Reply:
https://github.com/kubernetes-incubator/service-catalog/pull/1106#issuecomment-328223610